src/api/users.ts

```diff
@@ -12,3 +12,11 @@ type UserData = Pick<User, 'name' | 'email'>;
 export const createUser = async ({ email, name }: UserData) => {
   return client.post<User>('/users', { email, name });
 };
+
+export const getUserId = (user: User) => {
+  if (!user?.id) {
+    return 0;
+  }
+
+  return user.id;
+};
```

Comment:
But what the matter of this function? Why can't you just use user.id

Reply:
Done ✅